### PR TITLE
Make dashboard compatible with pipelines 0.6.0 

### PR DESCRIPTION
**Pull request by CarolynMabbott**
References issue #437 

# Changes

Updated pipeline.go to remove triggers and change some Params to ResourceParams which is required for pipelines v0.6.0 as well as change integration test back to pull from 0.6.0 pipelines
Also updated pipeline_test.go to reflect the changes in some Params to ResourceParams

Checked that creates PipelineRuns from both within the UI (creating a new pipelinerun and rebuild) and creating a pipelinerun via curls to the endpoint   
Checked that it correctly creates a PipelineRun from importing Tekton resources 
Checked that both PipelineRuns that succeeded and fail are listed correctly 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by steveodonovan** [comment]
All good from my side. @vtereso or @a-roberts might be best to review this

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ Tekton Dashboard is a general purpose, web-based UI for Tekton Pipelines. It all
 
 ## Pre-requisites
 
-[Tekton Pipelines](https://github.com/tektoncd/pipeline) 0.5.2 or later must be installed in order to use the Tekton Dashboard. Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
+[Tekton Pipelines](https://github.com/tektoncd/pipeline) 0.6.0 must be installed in order to use the Tekton Dashboard release 0.2.0 or master. Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
+To use Tekton Dashboard release 0.1.0 and 0.1.1 you must have [Tekton Pipelines](https://github.com/tektoncd/pipeline) 0.5.1 or 0.5.2 must be installed.
```
Comment (a-roberts) [inline README.md]: 0.5.1 won't work we have a pull request resource that's only in 0.5.2, let's point people to 0.1.1 and not 0.1.0 (0.1.0 is pointless now really)

**Comment by a-roberts** [comment]
/lgtm
/approve

/meow space

Thanks @CarolynMabbott ! Please announce this in tektoncd.slack.com

**Pull request merged**